index.bs

```diff
@@ -161,13 +167,16 @@ For uncalibrated magnetometer, the [=latest reading=] includes three [=map/entri
 and three additional [=map/entries=] whose [=map/keys=] are "xBias", "yBias", "zBias" and whose [=map/values=] contain the <a>hard iron distortion</a> correction around the 3 different axes.
 
 The sign of the <a>magnetic field</a> values must be according to the
-right-hand convention in a <a>local coordinate system</a> defined by the device.
+right-hand convention in a [=local coordinate system=] (see figure below).
+
 
-Note: The <dfn>local coordinate system</dfn> of a mobile device is usually defined relative to
-the device's screen when the device in its default orientation (see figure below).
+<img src="images/magnetometer_coordinate_system.svg" onerror="this.src='images/magnetometer_coordinate_system.png'" style="display: block;margin: auto;" alt="Magnetometer coordinate system.">
 
-<img src="images/magnetometer_coordinate_system.png" srcset="images/magnetometer_coordinate_system.svg" alt="Magnetometer coordinate system.">
+Reference Frame {#reference-frame}
+----------------
 
+The [=local coordinate system=] represents the reference frame for the [=sensor readings=].
```

Comment:
You want to agree on a consistent language with https://github.com/w3c/gyroscope/pull/29/files#diff-ec9cfa5f3f35ec1f84feb2e59686c34dR107

Reply:
done